tutorials/shading/shading_reference/shading_language.rst

```diff
@@ -187,6 +187,65 @@ Keep in mind that some architectures (mainly mobile) benefit a lot from this, bu
 to find out more. In all honesty though, mobile drivers are buggy, so, to stay out of trouble, make simple 
 shaders without specifying precision unless you *really* need to.
 
+Arrays
+------
+
+Arrays are containers for the multiple variables of similar type. In GLSL they can be declared local to function, global space or uniform.
```

Comment:
```suggestion
Arrays are containers for multiple variables of a similar type. In GLSL, they can be declared local to a function, global to a shader, or as an uniform.
```

PS: Should we mention GLSL functionality here, since only local arrays are implemented in Godot? This could be confusing to users.

Reply:
Removed